apps/nextra/pages/en/build/smart-contracts/book/structs-and-resources.mdx

```diff
@@ -222,7 +222,7 @@ module 0x2::m {
 
 ### Reading and Writing Fields
 
-If you need to read and copy a field's value, you can then dereference the borrowed field:
+If a field is copyable, you can read and copy a field's value by dereferencing the borrowed field:
```

Comment:
Can we update the below example as well?

Originally, to copy you had to do a `*&MyStruct` to copy, but that paradigm is long gone.  Additionally, I see in the example the `copy` keyword in `Bar { foo: copy foo }`, but that can be skipped correct?

Can we do something like:

```move
module 0x2::m {
  fun example() {
    let foo = Foo { x: 3, y: true };
    let bar = Bar { foo };
    let foo_ref = &foo;
    let bar_ref = &bar;
    let x: u64 = *foo_ref.x;
    let y: bool = *foo_ref.y;
    let foo2: Foo = *bar.foo;
  }
}
```

Reply:
Yes, that can be skipped. I kept it around as it did not pertain to this particular fix, but I can remove it.